src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -121,24 +123,29 @@ public static SyntaxNode RemoveMutant(SyntaxNode nodeToRemove)
             throw new InvalidOperationException($"Unable to find an engine to remove injection from this node: '{nodeToRemove}'");
         }
 
-        public static (string engine, int id) FindEngine(SyntaxNode node)
+        public static (int id, string engine, string type) FindAnnotations(SyntaxNode node)
```

Comment:
This seems somewhat similar to the rollback logic. Perhaps instead of a private class we should have a public class to hold this tuple data.